metrics/api/pom.xml

```diff
@@ -50,6 +50,14 @@
             <groupId>org.eclipse.microprofile.metrics</groupId>
             <artifactId>microprofile-metrics-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why is Micrometer an explicit dependency here? I thought the APIs should be MP and not Micrometere?

Reply:
Yes, not ideal, but we plan for Helidon to introduce its own neutral metrics API and, once that is in place, we will change our MP metrics implementation to  depend on that rather than directly on Micrometer.

But due to scheduling complications earlier in the spring and given our goal of having MP metrics support in M1 we decided to go ahead with a first MP implementation without waiting for the neutral API, knowing we would change the MP implementation once the neutral API is in place. 

The neutral API will likely be very close to the Micrometer API, partly for the convenience of developers and partly because of Micrometer's maturity in this space. In that case, the changes we will need to make in our MP implementation, although not zero, will be relatively small. Much of the code already uses a `MetricFactory` SPI abstraction to help minimize the changes required later.

We would definitely have preferred to have the neutral API in place first, but I just couldn't make that happen in time for MP metrics in M1.

Reply:
We should try to fix this for M2. It doesn't look very nice. I should be able to help Tim.

Reply:
Yes, we should implement our neutral API and adapt SE and MP to it soon.